src/scenes/index.js

```diff
@@ -1,30 +1,28 @@
 import React, {Component} from 'react'
-
-// Libraries
 import {
   BrowserRouter as Router,
-  Route
+  Route,
+  Switch
 } from 'react-router-dom'
-
-// Components
 import Home from './Home';
 import TradeHub from './TradeHub';
 
-class Root extends Component {
-  render() {
-    return (
-      <Router>
-        <div>
-          <Route exact path='/' render={(props) => (
-            <Home opts={this.props.opts} />
-          )}/>
-          <Route exact path='/tradehub' render={(props) => (
-            <TradeHub opts={this.props.opts} />
-          )}/>
-        </div>
-      </Router>
-    )
-  }
-}
-
-export default Root;
+export default class Root extends Component {
+  render = () => (
+    <Router>
+      <Switch>
+        <Route
+          exact
+          path="/"
+          render={(props) => <Home opts={this.props.opts} />}
```

Comment:
If an arrow function is in one line, you don't wrap the parameter in parentheses, so this should be:
```javascript
render={props => <Home opts={this.props.opts} />}
```